cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml

```diff
@@ -440,7 +441,7 @@ spec:
             - template
             type: object
         type: object
-    served: true
+    served: false
```

Comment:
What this changes from the current workflow?

Reply:
It disables the use of v1beta1 CRD version.